Activities/Directory.build.targets

```diff
@@ -1,23 +1,36 @@
 <Project>
-  <ItemGroup Condition="$(TargetFramework.StartsWith('net5'))"> 
-    <PackageReference Update="System.Activities" Version="5.0.0-20210615-01" />
-    <PackageReference Update="UiPath.Workflow" Version="5.0.0-20210615-01" />
-    <PackageReference Update="System.Activities.Core.Presentation" Version="5.0.0-20210724.1" />
-    <PackageReference Update="System.Activities.ViewModels" Version="1.0.0-20220307.1"/>
-  </ItemGroup>
-  <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp3.1'"> 
-    <PackageReference Update="System.Activities" Version="1.0.0-20201105-02" />
-    <PackageReference Update="UiPath.Workflow" Version="1.0.0-20201105-02" />
-    <PackageReference Update="System.Activities.Core.Presentation" Version="1.0.0-alpha003" />
-  </ItemGroup>
-  <ItemGroup>
-	<PackageReference Update="UiPath.Robot.Api" Version="22.4.1" PrivateAssets="All" />
-	<PackageReference Update="UiPath.Activities.Api" Version="22.4.1" PrivateAssets="All" />
-	<PackageReference Update="UiPath.Activities.Contracts" Version="22.4.1" PrivateAssets="All" />
-	<PackageReference Update="UiPath.Telemetry.Client" Version="1.3.18" PrivateAssets="All" />
-	<PackageReference Update="UiPath.Platform" Version="22.4.1"  PrivateAssets="All" />
-  </ItemGroup>
-  <PropertyGroup>
-    <DefineConstants Condition="'$(TargetFramework)' == 'net5.0'">$(DefineConstants);NETSTANDARD</DefineConstants>
-  </PropertyGroup>
+	<ItemGroup Condition="$(TargetFramework.StartsWith('net6'))">
+		<PackageReference Update="Uipath.Workflow.Runtime" Version="6.0.0-20220401-03" />
+		<PackageReference Update="UiPath.Workflow" Version="6.0.0-20220401-03" />
+		<PackageReference Update="System.Activities.Core.Presentation" Version="6.0.0-20220506.1" />
+		<PackageReference Update="System.Activities.ViewModels" Version="1.0.0-20220422.1"/>
+		<PackageReference Update="System.Management.Automation" Version="7.2.5" />
+		<PackageReference Include="Microsoft.PowerShell.Commands.Diagnostics" Version="7.2.5"/>
+		<PackageReference Include="Microsoft.PowerShell.ConsoleHost" Version="7.2.5" />
+		<PackageReference Include="Microsoft.PowerShell.Commands.Utility" Version="7.2.5" />
+		<PackageReference Include="Microsoft.PowerShell.Commands.Management" Version="7.2.5" />
+		<PackageReference Include="Microsoft.WSMan.Management" Version="7.2.5" />
+		<PackageReference Include="Microsoft.CodeAnalysis.VisualBasic" Version="4.0.1" />
+	</ItemGroup>
+	<ItemGroup Condition="$(TargetFramework.StartsWith('net5'))">
```

Comment:
The Scripting package should support .NET6 only.

Reply:
This affects ALL packages and is left here for the ones that still target net5, like Database and others. This should not be changed, because the other packages will no longer build properly.